src/main/java/org/openelisglobal/patient/controller/rest/PatientManagementRestController.java

```diff
@@ -216,4 +216,43 @@ private Patient loadForUpdate(PatientManagementInfo patientInfo) {
         return patient;
     }
 
+    private void validatePatientInfo(Errors errors, PatientManagementInfo patientInfo) {
```

Comment:
Thanks @common-cold , can you just update the existing functionality rather than commenting it out ??